### ISSUE TITLE
Mermaid %%init{ ... }%% does not work

### ISSUE THREAD

**Issue by nanoant** [typora/typora-issues#5291]
**Describe the bug**
Typora Mermaid block ignore `%%init{ ... }%% clauses which prevents graph customization. This customization was introduced in [v8.7.0 (2020)](https://github.com/mermaid-js/mermaid/blob/develop/docs/CHANGELOG.md)

**To Reproduce**
Use following Markdown block:
```markdown
~~~mermaid
%%{init: { "theme": "neutral", "themeVariables": { "fontSize": "32px", "fontFamily": "Times" } } }%%
flowchart LR
 A-->B
~~~
```
Typora shows graph with no customizations:
![image](https://user-images.githubusercontent.com/103067/173138089-a3a0f984-e33f-4736-8dd2-6471e2be2524.png)

**Expected behavior**
[Mermaid Live editor](https://mermaid-js.github.io/mermaid-live-editor/) shows customizations properly:
![image](https://user-images.githubusercontent.com/103067/173138249-a5dc4149-94f6-41e0-a885-b466a3d8391f.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 and macOS 12.4

**Typora Version**
1.3.3-dev (also latest stable)

**Additional context**
I suspect this may be related with Mermaid customizations code using `--mermaid` CSS rules in Typora `user.base.css`.

**Comment by abnerlee** [comment]
put into  #1661

**Comment by abnerlee** [comment]
fixed in 1.4.4

**Comment by SamuelMiller** [comment]
In Catalina OS, Typora's (Version 1.5.8 (6707)) Mermaid implementation is unresponsive to custom changes in font size:
%%{init: { "theme": "neutral", "themeVariables": { "fontSize": "32px", "fontFamily": "Times" } } }%%

Font family changes but not font size.
